docs/UserGuide.md

```diff
@@ -117,23 +117,45 @@ Examples:
 * `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 * `edit 3 sch/NJC stream/` Edits the school of the 3rd person to be `NJC` and clears academic stream data.
 
-### Locating persons by name: `find`
+### Finding a person by fields: `find`
 
-Finds persons whose names contain any of the given keywords.
+Finds persons whose fields match the given keywords.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `find [n/KEYWORD …] [a/KEYWORD …] [e/KEYWORD …] [p/KEYWORD …] `
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+* There should be at least one field i.e. `find` is an invalid command.
+* The search is case-insensitive. e.g.`hans` will match `Hans`.
+* The order of the keywords does not matter. e.g. `Clementi West` will match `West Clementi`.
+* The keyword does not need to match the field exactly. e.g. `john@gmail.com` will match `leejohn@gmail.com`.
+* A field just needs to match at least one keyword. e.g. `Hans Bo` will match `Hans Gruber`, `Bo Yang`.
+* All fields specified need to match for person to be matched.<br>
+  e.g. `find n/john a/Clementi`
+    * will match person named `john` with address `West Clementi Street`.
+    * will not match person named `john` with address `Bedok Reservoir`.
```

Comment:
Will there be multiple matches i.e. person(s)?

Reply:
yes will return multiple result.  Here i'm just giving one example, is it clear?

Reply:
I think comprehension is subjective :0 
I personally thinks it could be clearer? Maybe either with a disclaimer at the end of line 132 or after the example?

Reply:
ok I'll change it a bit

Reply:
subsequently, i might add a screenshot to make it clearer